src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -108,11 +114,10 @@ public void loadExistingCredentials() {
       loadAppDefaultCredentials();
       return;
     }
-    try (InputStream inputStream =
-        User.class.getClassLoader().getResourceAsStream(CLIENT_SECRET_FILENAME)) {
-      googleCredentials =
+    try {
+      terraCredentials =
           GoogleOauth.getExistingUserCredential(
```

Comment:
I was thinking about ways to make this more OOPy, and maybe an object representing the user's auth "stuff" would make sense. It could change whenever we switch servers, but be outlived by the `User` object. It could also implement an interface that abstracts over whether it's using a JWT or an access token.

Reply:
Moved the client secrets into GoogleOauth.  I think there's more refactoring we could do per Jay's comment, but I think I'd like to leave that for a separate effort.